frontend/packages/knative-plugin/src/components/overview/FilterTable.scss

```diff
@@ -0,0 +1,10 @@
+.kn-filter-table {
+  .kn-filter-table {
```

Comment:
do we need this? parent child both have same class

Reply:
This is to increase the priority of the selector
cc: @vikram-raj

Reply:
Thanks , got it :)